aperturedb/ConnectionLoader.py

```diff
@@ -76,14 +85,19 @@ def validate(self):
 class ConnectionLoader(ParallelLoader.ParallelLoader):
     """**ApertureDB Connection Loader.**
 
-        This class is to be used in combination with a "generator".
-        The generator must be an iterable object that generated "entity_data"
-        elements.
+    This executes in conjunction with a **generator** object for example :class:`~aperturedb.ConnectionLoader.ConnectionGeneratorCSV`,
+    which is a class that implements iterable inteface and generates "connection data" elements.
 
     Example::
 
-            entity_data = {
+            connection_data = {
                 "class":       connection_class,
+                "ref1_class": "_Image",
```

Comment:
we should probably rename ref1 and ref2 as "src" and "dst", respectively, at some point to make it more explicit and consistent with our API

Reply:
will do.